### PR TITLE
New Cheat Sheet for Mongo Shell

### PR DESCRIPTION
**Pull request by riteshshrv**
**What does your Instant Answer do?**
Shows quick reference of Mongo Shell commands

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Shows the most common commands used in Mongo Shell for MongoDB

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://docs.mongodb.org/manual/reference/mongo-shell/

**Why did you choose this data source?**
Its the official documentation

**Are there any other alternative (better) data sources?**
May be but this one is the official source

**What are some example queries that trigger this Instant Answer?**
mongo shell commands, mongodb cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
MongoDB users

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
DuckDuckGo

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screenshot from 2015-11-08 03 01 34
screenshot from 2015-11-08 03 01 55

## Checklist
Please place an 'X' where appropriate.

```
- [] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 10
        - [x] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 11
        - [x] Microsoft edge
        
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Linux
        - [x] Google Chrome
        - [x] Firefox
        
    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

**Comment by daxtheduck** [comment]
@riteshshrv Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/mongo-shell.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
+{  
+   "id":"mongo_shell_cheat_sheet",
+   "name":"Mongo Shell Quick Reference",
+   "description":"Mongo is an interactive JavaScript shell interface to MongoDB which is a NoSQL language for accessing and manipulating databases.",
+   "metadata":{  
+      "sourceName":"mongodb",
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/mongo-shell.json]: case this to ```mongoDB``` or ```MongoDB```

File: share/goodie/cheat_sheets/json/mongo-shell.json [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      "sourceUrl":"https://docs.mongodb.org/manual/reference/mongo-shell/"
+   },
+   "template_type":"reference",
+   "section_order":[  
+      "Command Line Options",
+      "Command Helpers",
+      "Basic Shell JavaScript Operations",
+      "Queries"
+
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/mongo-shell.json]: Remove this extra new line.

File: share/goodie/cheat_sheets/json/mongo-shell.json [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+      "Command Line Options",
+      "Command Helpers",
+      "Basic Shell JavaScript Operations",
+      "Queries"
+      
+   ],
+   "aliases":[  
+      "mongo statements",
+       "mongo",
+       "mongodb",
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/mongo-shell.json]: Can you fix the indentation on this list?

File: share/goodie/cheat_sheets/json/mongo-shell.json [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+     	 {  
+        	"key":"help",
+        	"val":"Show help"
+ 	 	 },
+     
+	     {  
+	        "key":"db.help()",
+	        "val":"Show help for database methods"
+	     },
+
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/mongo-shell.json]: Your cheat sheet have indentation problem, Use tools like ```jsonformatter``` to fix that.

**Comment by moollaza** [comment]
Looks like this is a duplicate of #1636. @riteshshrv if you'd like to collaborate on that one, or wait until it's merged and submit updates, that would be ideal. I'll close this one and merge #1636 once it's ready

**Pull request closed without merging**